packages/react-scripts/fixtures/monorepos/lerna-ws/bootstrap.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const { execSync } = require('child_process');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+const { platform } = require('os');
+const crossSpawn = require('cross-spawn');
+
+// function shouldUseYarn() {
+//   try {
```

Comment:
What’s up with this?

Reply:
That is to force npm as the client (as also specified in lerna.json) so that lerna bootstrap --hoist works.
Sorry, forgot to clean that up.  But, it's good you caught it because it raises a good question about the recommended approaches.

This test case is hoisting react which allows the shared components to pick it up without actually specifying it as a dependency -- which is handy, but probably bad and will be broken by https://github.com/facebook/create-react-app/pull/1752.  I originally did that to avoid components having to specify react and match version with the app for fear that mismatched versions would cause issues.  (Note: YW automatically hoists, so this is partially matching that behavior.)

But maybe this should be approached some other way.  Ideas?  Is it ideal for shared components to specify react as a peerDependency and match the app react version?